src/components/pages/browser-not-supported.yml

```diff
@@ -0,0 +1,7 @@
+headerText:
+ en: We have detected that your browser version is not supported by our website
+ de: Wir haben festgestellt, dass Ihre Browserversion von unserer Website nicht unterstützt wird
+mainText:
+ en: Please download a modern broswer of your choice, some examples below
+ de: Bitte laden Sie einen modernen Browser Ihrer Wahl herunter, einige Beispiele unten
+ 
```

Comment:
This is weird. On local I have only one line break at the end which I uploaded when it was complaining about a missing line break. Now it looks like this, I don't know why....

Reply:
The last line is not empty (it contains a single space), but also missing the final newline. If you install the editorconfig plugin, it will make sure this doesn't happen.

Reply:
Geinstalled, danke! It seems to work